FirelandsProject/firelands-cata
# Correct creature data & class stat levels

### Changes proposed

- Correct creature stat data, cause ~90% of them are wrong.
- Update all creature stat data that are values of 1.

### Preview Current Issues:

1. With a unchanged database, create and/or log into a Blood Elf character
2. Check the Sunstrider Guardians
3. See their health reflects their health modifier of 15.

**Note:** This is only one of the issues that is noticeable. Another example is below.

Incorrect:
![image](https://user-images.githubusercontent.com/5354732/189158014-ccf191f7-7ead-4ca6-bd3c-05e4fb3b80fa.png)

Correct:
![image](https://user-images.githubusercontent.com/5354732/189158132-60470458-eb96-4998-8601-a7df70efd068.png)

### How to Test the Changes:


1. Update your database with the new SQL file
2. Login into your Blood Elf character
3. Check the health on the Sundstrider Guardian.

### Known issues and TODO list


- [ ] Finish researching and calculating missing data for creatures.
- [ ] Update creatures with the correct expansion level and modifiers.

---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
